worlds/soe/__init__.py

```diff
@@ -272,6 +306,10 @@ def generate_basic(self):
         wings_item = self.create_item('Wings')
         self.world.get_location(wings_location, self.player).place_locked_item(wings_item)
         self.world.itempool.remove(wings_item)
```

Comment:
didn't notice this last time, but should probably avoid adding the item to the itempool instead of later scanning for and removing it.

Reply:
This was added as a hotfix to avoid a softlock. And then I copied it for energy core :S
Will fix both in a couple of minutes.